### ISSUE TITLE
Uncaught TypeError: foldFinder.findFoldMarkers is not a function

### ISSUE THREAD

**Issue by averrin** [t9md/atom-lazy-motion#15]
Just call lazy-motion.

**Atom**: 1.14.0-beta1 x64
**Electron**: 1.3.13
**OS**: Ubuntu 16.04.1
**Thrown From**: [lazy-motion](https://github.com/t9md/atom-lazy-motion) package 0.5.1

### Stack Trace

Uncaught TypeError: foldFinder.findFoldMarkers is not a function

```
At /home/alexeynabrodov/.atom/packages/lazy-motion/lib/utils.coffee:11

TypeError: foldFinder.findFoldMarkers is not a function
    at saveEditorState (/packages/lazy-motion/lib/utils.coffee:11:30)
    at Object.start (/packages/lazy-motion/lib/main.coffee:56:29)
    at /packages/lazy-motion/lib/main.coffee:22:34)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -0:14.6.0 editor:indent (input.hidden-input)
  7x -0:13.1.0 core:backspace (input.hidden-input)
     -0:11.7.0 snippets:next-tab-stop (input.hidden-input)
     -0:11.7.0 snippets:expand (input.hidden-input)
     -0:11.7.0 editor:indent (input.hidden-input)
     -0:10.6.0 core:backspace (input.hidden-input)
     -0:09.7.0 vim-mode-plus:activate-normal-mode (input.hidden-input)
  8x -0:09.4.0 vim-mode-plus:move-up (input.hidden-input)
     -0:05.4.0 lazy-motion:forward (input.hidden-input)
     -0:05.2.0 vim-mode-plus:mark (input.hidden-input)
     -0:05.1.0 vim-mode-plus:set-input-char-e (input.hidden-input)
  2x -0:04.8.0 lazy-motion:forward (input.hidden-input)
     -0:04.6.0 vim-mode-plus:insert-after (input.hidden-input)
     -0:00.2.0 blur (input.hidden-input)
     -0:00.2.0 vim-mode-plus:reset-normal-mode (atom-text-editor.editor.vim-mode-plus.is-focused.insert-mode)
     -0:00.2.0 blur (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
```

### Non-Core Packages

```
accents-ui 1.2.0 
advanced-open-file 0.16.5 
atom-beautify 0.29.16 
atom-toolbar 0.1.6 
autocomplete-paths 1.0.5 
browser-refresh 0.9.0 
build 0.67.0 
build-gulp 0.11.0 
builder 0.0.1 
busy 0.7.0 
choose-pane 0.5.1 
color-picker 2.2.4 
colorful-cursor 0.3.1 
cursor-history 0.9.1 
dartlang 0.6.45 
editorconfig 2.2.0 
emmet 2.4.3 
environment 1.3.1 
file-icons 2.0.9 
find-and-replace-under-cursor 0.3.0 
firacode 0.1.0 
firewatch-syntax 1.1.1 
geiger 2.1.0 
git-change-marker 0.11.0 
git-checkout 0.3.2 
git-control 0.8.2 
git-log 0.4.1 
go-debug 1.0.3 
go-find-references 1.2.3 
go-oracle 0.2.1 
go-outline 0.6.2 
go-plus 5.1.0 
go-signature-statusbar 1.2.0 
javascript-snippets 1.2.1 
jumpy 3.1.3 
language-babel 2.49.4 
language-diff 0.7.0 
language-lua 0.9.11 
lazy-motion 0.5.1 
less-than-slash 0.16.0 
linter 1.11.21 
linter-eslint 8.0.0 
linter-govet 0.1.5 
linter-pep8 1.3.2 
merge-conflicts 1.4.4 
minimap 4.26.6 
minimap-bookmarks 0.4.2 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.1 
minimap-git-diff 4.3.1 
minimap-hide 0.3.0 
minimap-highlight-selected 4.5.0 
minimap-pigments 0.2.2 
narrow 0.7.1 
nerd-treeview 0.4.3 
neutron-syntax 0.7.0 
paner 0.4.0 
pigments 0.38.0 
project-find-navigation 0.3.0 
project-folder 1.2.1 
project-plus 0.9.0 
project-switcher2 1.3.2 
protector 0.2.2 
pull-request 0.2.1 
python-import-magic 0.8.0 
python-tools 0.6.9 
recent-finder 0.5.0 
remember-file-positions 0.2.2 
resonance-syntax 0.2.0 
resonance-ui 0.11.0 
seti-syntax 1.0.1 
swackets 0.12.1 
sync-settings 0.8.1 
synced-sidebar 0.4.3 
tester-go 2.0.2 
time-status 1.1.2 
underline-trailing-whitespace 1.4.0 
unsaved-changes 0.1.0 
vim-mode-plus 0.80.0 
vim-mode-plus-move-to-symbols 0.2.0 
vim-mode-plus-project-find-from-search 0.2.0 
vim-mode-plus-subword-movement 0.2.2 
watson 0.2.4 
Zen 0.16.4 
zentabs 0.8.8 
```

**Comment by t9md** [comment]
Fixed. Released as v0.5.2 now.

FYI
If you use vim-mode-plus 0.80.0 and later, you don't need.
- vim-mode-plus-project-find-from-search: It's merged into vmp-core.(`enter` in search-form)
- vim-mode-plus-subword-movement: It's merged into vmp-core.(no default keymap)
- Zen: Unless you need Zen specific feature, vmp's `cmd-enter`(`maximize-pane`) can use for that.

**Comment by averrin** [comment]
thx, i use vmp feature instead zen=)
